### PR TITLE
feature(content-band-list-spacing-issue): Content band list spacing issue.

### PR DESCRIPTION
**Pull request by nimithshetty17**
## Issue

Fixes #[MERATIVE-862](https://jira.sdlc.merative.com/browse/MERATIVE-862)

## Description

When authoring the product family pages and adding the content band list with icons, there is no spacing/padding between the first and second row if more than three cards are added into this section.

**Changed**

image

image

- Figma Link - https://www.figma.com/file/vw24IPXXt4vCh9BNumka9A/Web--Merative-Digital-Design-System-2.0?type=design&node-id=1459-14993&mode=design&t=BbstBwzZqcXHN2Aw-0

## Test URLs
  
- Before (Changes from `main`): https://main--merative2--hlxsites.hlx.page/merge-imaging
- After (Changes from this PR): https://content-band-list-spacing-issue--merative2--nimithshetty17.hlx.page/merge-imaging
  
## Testing Instruction

To test the content band list spacing and its responsiveness with respect to the design across all devices.

### CODE REVIEW

**Review by sharathmrft — CHANGES_REQUESTED** [review]
@nimithshetty17 Spacing 48 px is fixed and appearing correctly but there is a misplacement of Image in mobile device as shown below.
image

@proeung

**Comment by nimithshetty17** [comment]
@sharathmrft - The mobile icon alignment issue is fixed.

**Review by sharathmrft — DISMISSED** [review]
@nimithshetty17 Image is not aligned properly
MI1
@proeung

**Comment by nimithshetty17** [comment]
@sharathmrft - Image alignment issue is fixed.

**Comment by sachinmesh** [comment]
Checking the content band only, the watch video CTA play icon is wrong 
image

Rest looks good to me.

@proeung @nimithshetty17 @sharathmrft

File: blocks/columns/columns.js [hunk 1]
```diff
@@ -1,6 +1,10 @@
 export default function decorate(block) {
+  const numberOfChildren = block.childElementCount || 0;
   const cols = [...block.firstElementChild.children];
   block.classList.add(`columns-${cols.length}-cols`);
+  if (numberOfChildren > 1) {
+    block.classList.add('add-bottom-space');
+  }
```
Comment (proeung) [inline blocks/columns/columns.js:7]: @nimithshetty17 Is there a reason why this class (`add-bottom-space`) is being added using JavaScript? Ideally, you can apply this fix in just CSS.

```
.columns.content-band.icon-list > div {
  margin-bottom: var(--spacer-layout-04);
}
```

File: blocks/columns/columns.css [hunk 2]
```diff
@@ -100,6 +100,18 @@ main .columns-container h4 {
   margin-bottom: 0;
 }
 
+.columns.content-band.icon-list.add-bottom-space > div {
+  margin-bottom: var(--spacer-layout-04);
+}
```
Comment (proeung) [inline blocks/columns/columns.css:105]: bottom spacing should exist for each row, meaning that there's no need to have a class for just applying the bottom spacing.

```suggestion
.columns.content-band.icon-list > div {
  margin-bottom: var(--spacer-layout-04);
}
```

File: blocks/columns/columns.css [hunk 3]
```diff
@@ -100,6 +100,18 @@ main .columns-container h4 {
...
+
+.columns.content-band.icon-list.add-bottom-space > div > div > p {
+  justify-content: left;
+}
```
Comment (proeung) [inline blocks/columns/columns.css:109]: This is not needed as all picture/img within each row of the block should be aligned left.

.columns.content-band > div > div > p.only-picture {
  justify-content: left;
}

```suggestion
```

**Review by proeung — CHANGES_REQUESTED** [review]
@nimithshetty17 Looks good, but I have some code changes/suggestions.

@sachinmesh Thanks for conducting design QA on this PR. As for the "Play Icon" display issue that you discovered. This bug will be addressed in this ticket (https://jira.sdlc.merative.com/browse/MERATIVE-858). We can keep this PR to fixing the spacing issue for each row within this column block pattern.

File: blocks/columns/columns.css (hunk 3 above)
Comment (nimithshetty17) [reply blocks/columns/columns.css]: @proeung - have made these changes.

**Review by sachinmesh — APPROVED** [review]
Checked only the content band list spacing, when there are more than 3 cards. Good to go.

@proeung @nimithshetty17 @sharathmrft

**Review by proeung — APPROVED** [review]
LGTM!

**Pull request merged**